in_text/text_01_spatial.yml

```diff
@@ -97,7 +97,7 @@ entities:
         This is the ID named seg_id_nat from the Geospatial Fabric for National Hydrologic Modeling.
         This feature corresponds to "seg_id_nat" in other data files. There are 70 unique identifiers in this field.
       attr-defs: >-
-        https://www.sciencebase.gov/catalog/item/535eda80e4b08e65d60fc834
+        http://dx.doi.org/doi:10.5066/F7542KMD
```

Comment:
In first data release, this link was modified by Sam to be the DOI that corresponds to the citation for clarity.